src/GitHub.InlineReviews/Commands/IVsCommand.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Threading.Tasks;
+
+namespace GitHub.InlineReviews.Commands
+{
+    /// <summary>
+    /// Represents a Visual Studio command that does not accept a parameter.
+    /// </summary>
+    public interface IVsCommand : IVsCommandBase
```

Comment:
This framework for VS commands is similar to the one that we already have in GitHub.VisualStudio for these commands: https://github.com/github/VisualStudio/tree/master/src/GitHub.VisualStudio/Menus.

I used a different framework however because I need to support command arguments. It might be worth merging these two frameworks in future.